net/ovsd/Makefile

```diff
@@ -0,0 +1,43 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ovsd
+PKG_VERSION:=2020-10-12
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://gitlab.hhi.fraunhofer.de/wn-ina/ovsd.git
+PKG_SOURCE_VERSION:=e6a07f27dd8a5893eb6110c69922077995b9b26a
```

Comment:
Why are you using old version? Can not we use the latest one?
In the suggestion, if you will approve it, it is updated to the latest version.